xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -185,8 +185,7 @@ bool CDVDDemuxFFmpeg::Aborted()
   if(m_timeout.IsTimePast())
     return true;
 
-  CDVDInputStreamFFmpeg * input = dynamic_cast<CDVDInputStreamFFmpeg*>(m_pInput);
-  if(input && input->Aborted())
+  if (m_pInput && m_pInput->IsStreamType(DVDSTREAM_TYPE_FFMPEG) && static_cast<CDVDInputStreamFFmpeg*>(m_pInput)->Aborted())
```

Comment:
right, but this is after avformat_close_input is called. avformat must not be open without an underlying input stream. this case seems to be related to the otherstream hack of pvr.